js/main.js

```diff
@@ -375,6 +380,7 @@ $(function() {
   //End Game
   //Get the winner
   function getWinner(scoreP1, scoreP2) {
+    debugger;
```

Comment:
Need to remove Debugger!

Reply:
Hi Richard,

I removed the debugger as requested.

Thank you